lib/std/os/bits/linux/mips.zig

```diff
@@ -383,9 +383,48 @@ pub const SYS = extern enum(usize) {
     statx = Linux + 366,
     rseq = Linux + 367,
     io_pgetevents = Linux + 368,
+    semget = Linux + 393,
+    semctl = Linux + 394,
+    shmget = Linux + 395,
+    shmctl = Linux + 396,
+    shmat = Linux + 397,
+    shmdt = Linux + 398,
+    msgget = Linux + 399,
+    msgsnd = Linux + 400,
+    msgrcv = Linux + 401,
+    msgctl = Linux + 402,
+    clock_gettime64 = Linux + 403,
+    clock_settime64 = Linux + 404,
+    clock_adjtime64 = Linux + 405,
+    clock_getres_time64 = Linux + 406,
+    clock_nanosleep_time64 = Linux + 407,
+    timer_gettime64 = Linux + 408,
+    timer_settime64 = Linux + 409,
+    timerfd_gettime64 = Linux + 410,
+    timerfd_settime64 = Linux + 411,
+    utimensat_time64 = Linux + 412,
+    pselect6_time64 = Linux + 413,
+    ppoll_time64 = Linux + 414,
+    io_pgetevents_time64 = Linux + 416,
+    recvmmsg_time64 = Linux + 417,
+    mq_timedsend_time64 = Linux + 418,
+    mq_timedreceive_time64 = Linux + 419,
+    semtimedop_time64 = Linux + 420,
+    rt_sigtimedwait_time64 = Linux + 421,
+    futex_time64 = Linux + 422,
+    sched_rr_get_interval_time64 = Linux + 423,
+    pidfd_send_signal = Linux + 424,
     io_uring_setup = Linux + 425,
     io_uring_enter = Linux + 426,
     io_uring_register = Linux + 427,
+    open_tree = Linux + 428,
+    move_mount = Linux + 429,
+    fsopen = Linux + 430,
+    fsconfig = Linux + 431,
+    fsmount = Linux + 432,
+    fspick = Linux + 433,
+    pidfd_open = Linux + 434,
+    clone3 = Linux + 435,
```

Comment:
you missed `close_range` which is `+ 436`

also while you're in here:
```
439     n32     faccessat2                      sys_faccessat2
440     n32     process_madvise                 sys_process_madvise
```

Reply:
Yes, I was wondering what the gap at `436` was. 😄

`close_range` is not in any of our other syscall number lists in the `os/bits/linux` files?

It's been added here now, along with `faccessat2` and `process_madvise`.

Reply:
`close_range` is very new.